apps/example/src/commands/ping.ts

```diff
@@ -8,6 +7,28 @@ export default {
 	async run(interaction) {
 		return void interaction.reply({
 			content: 'hello world!!!!!!!!',
+			components: [
+				{
+					type: 1,
+					components: [
+						{
+							type: 2,
+							label: 'Click me!',
+							style: 1,
+							custom_id: `ping-${interaction.user.id}`,
+						},
+						{
+							type: 2,
+							label: 'i have no params',
+							style: 2,
+							emoji: {
+								name: "🫢"
```

Comment:
Slight note that this character doesn't render on some clients, most notably Windows 11.

Reply:
Not that big of a problem, the emoji was added mainly for fun.